src/server/routes.js

```diff
@@ -3,6 +3,7 @@ const routes = require('next-routes')();
 routes
   .add('/', '/home/index')
   .add('/signin', '/auth/signin')
-  .add('/signup', '/auth/signup');
```

Comment:
if we only redirect /auth/* pages, then we can remove these alias path.
Cause these will affect `pathname` and `asPath` when using `useRouter`.

Reply:
i suggest we keep it for now, we could always open another issue if we want to remove alias path.